packages/configuration-development/package.json

```
@@ -6,7 +6,7 @@
     "@hint/formatter-summary": "^2.0.1",
     "@hint/hint-axe": "^2.1.1",
     "@hint/hint-babel-config": "^2.1.3",
-    "@hint/hint-button-type": "^1.0.1",
+    "@hint/hint-button-type": "2.0.0",
```

This and others should have `^` (I put another comment where this should be fixed in the script).
```suggestion
    "@hint/hint-button-type": "^2.0.0",
```